presto-docs/src/main/sphinx/release/release-0.290.rst

```diff
@@ -0,0 +1,125 @@
+* Upgrade avro to version 1.11.4. :pr:`23868`
+* Upgrade commons-codec to version 1.17.0. :pr:`23868`
+* Upgrade commons-compress to version 1.26.2. :pr:`23868`
+* Upgrade commons-io to version 2.16.0. :pr:`23794`
+* Upgrade commons-io to version 2.16.1. :pr:`23868`
+* Upgrade commons-lang3 to version 3.14.0. :pr:`23868`
+* Upgrade protobuf-java to version 3.25.5. :pr:`23797`
+* Upgrade protobuf-java-util to version 3.25.5. :pr:`23797`
+* Upgrade to centralize dependency versions in the parent pom.xml from presto-cassandra/pom.xml for consistency and ease of maintenance.  :pr:`23818`
+
+Presto C++ (Native Execution) Changes
+______________________________________
+* Fix ``task.writer-count`` and ``task.partitioned-writer-count`` configuration properties in Presto C++ for consistency with Presto. :pr:`23902`
+* Fix a bug where users weren't able to set the ``native_expression.max_array_size_in_reduce`` session property. :pr:`23856`
+* Fix plan validation failures for some join queries running with spill enabled when using Presto C++. :pr:`23595`
+* Improve I/O and CPU usage by merging FilterNode into TableScanNode where possible. Also, added ``$path`` and ``$bucket`` to split info, and fixed the split counts in the coordinator UI. :pr:`23755`
+* Improve arbitrator configs to use the new string-based format. :pr:`23496`
+* Improve setting buffer chunk size in ChunkedSliceOutput, which can reduce memory in the cases of large number of small output buffers. :pr:`23707`
+* Add test for logical type storage in Parquet files. :pr:`23388`
+* Add a metric ``presto_cpp.memory_pushback_expected_reduction_bytes`` to track expected reduction in memory after a pushback attempt. :pr:`23872`
+* Add a new counter, presto_cpp.memory_pushback_reduction_bytes, to monitor the actual memory reduction achieved with each memory pushback attempt. :pr:`23813`
+* Add ``native_max_local_exchange_partition_count`` session property which maps to the ``max_local_exchange_partition_count`` velox query property introduced in facebookincubator/velox#11292. :pr:`23910`
+* Add session property: ``native_writer_flush_threshold_bytes`` which specifies the minimum memory footprint size required to reclaim memory from a file writer by flushing its buffered data to disk. :pr:`23891`
+* Add session property: ``native_max_page_partitioning_buffer_size`` which specifies the maximum bytes to buffer per PartitionedOutput operator to avoid creating tiny SerializedPages. :pr:`23853`
+* Add session property: ``native_max_output_buffer_size`` which specifies the maximum size in bytes for the task's buffered output. The buffer is shared among all drivers. :pr:`23853`
+* Add incremental periodic cache persistence for Presto C++ worker. :pr:`23626`
+* Add native system session property provider. :pr:`23045`
+* Remove session property ``native_join_spiller_partition_bits``. :pr:`23906`
+* Reverted merging of FilterNode into TableScanNode done in pr `23755`. :pr:`23755`
```

Comment:
```suggestion
* Revert merging of ``FilterNode`` into ``TableScanNode`` done in :pr:`23755`. :pr:`23855`
```
Nit of formatting, fix link to PR 23755, correct end link to PR 23855. 
 
Note: I'll mention that "Revert" is not in the [Order of changes](https://github.com/prestodb/presto/wiki/Release-Notes-Guidelines#order-of-changes) in the Release Notes Guidelines, which may mean that we need to add "Revert" to the Order of changes.